flask_cors/__init__.py

```diff
@@ -544,3 +561,12 @@ def _serialize_options(options):
 
     if isinstance(options.get('max_age'), timedelta):
         options['max_age'] = str(int(options['max_age'].total_seconds()))
+
+
+def getLogger(app=None):
```

Comment:
In general this function is OK. I would suggest considering splitting it into 2 functions:

```python
def getLoggerNamePrefix(app=None):
    if app: # favor app, if given
        return app.logger_name
    try:
        return current_app.logger_name
    except RuntimeError: # "working outside app context"
        return 'flask.ext'

def getLogger(app=None):
    return logging.getLogger('%s.cors' % getLoggerNamePrefix(app))
```